components/backup/base/privatelink/backup-tenants-schedule.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: velero.io/v1
```

Comment:
This one looks very similar to the schedule we use on public clusters, can we reuse the same one?

Reply:
I checked and they are exactly the same. I have a proposal to remove duplication, let me open another PR with one commit on top of yours for you to see. If you agree with the change, you can apply the changes in this PR.

Reply:
here it is: https://github.com/redhat-appstudio/infra-deployments/pull/3315

Reply:
thanks @hugares, added what you suggested.